wordpress-mobile/WordPress-iOS
# What's New: Add chrome material background to the "Continue" button

Fixes #14720

Adds a chrome material background to the view containing the "Continue" button, and moves the table view behind that view to emphasize the blur effect (see example screenshots below):

**Note:** 
- **I added a few more fake announcements to emphasize the scrolling. These will all go away once the real data come in.**

Chrome Light | Chrome Dark
--- | ---
<img height="500" alt="chrome light" src="https://user-images.githubusercontent.com/34376330/91080745-bab9f800-e60b-11ea-8a80-c198f8dfabf9.png"> | <img height="500" alt="chrome dark" src="https://user-images.githubusercontent.com/34376330/91080833-dc1ae400-e60b-11ea-825e-f060023f2e35.png">

To test:

- Go to Me/App Settings/What's New in Wordpress
- Make sure that the background of the view containing the "Continue" button matches the designs in #14720 and the sample screens above, and that the subtle blur effect is visible.
- Try different devices/simulators and orientations, and make sure that the views look right.

PR submission checklist:

- [x] ~~I have considered adding unit tests where possible.~~ Future PR
- [x] ~~I have considered adding accessibility improvements for my changes.~~ Future PR
